rsocket-core/src/commonMain/kotlin/io/rsocket/kotlin/connection/LoggingConnection.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.rsocket.kotlin.connection
+
+import io.ktor.utils.io.core.*
+import io.rsocket.kotlin.frame.*
+import kotlinx.coroutines.*
+
+class LoggingConnection(private val delegate: Connection) : Connection {
+    override val job: Job get() = delegate.job
+
+    private fun log(tag: String, packet: ByteReadPacket) {
+        println("\n$tag: ${packet.dumpFrameToString()}")
```

Comment:
System.err would be my preference or pipe through java.util.logging.  What is the logging API for rsocket-kotlin?

Reply:
For now, there is no standard multiplatform logging library exist as I know. 
But I found one, that works everywhere - https://github.com/touchlab/Kermit. Touchlab is now one of the biggest companies, who are creating libraries for multiplatform, so I was thinking about using their lib.